src/cljs/token/wallets/page.cljs

```diff
@@ -43,8 +46,21 @@
        (doall (map (fn [id]
                      ^{:key id}
                      [:div [wallet id]])
-                   @accounts))]]
-     [:div.wallet-actions
-      [:a.trade-eth {:href "#"} "Trade ETH"]
-      [:a.create-wallet {:href "#"}
-       [:span.add-wallet] "Create a wallet"]]]))
+                   @accounts))]
+      [:a.wallet-btn {:href (str "#/")}
+       [:div.wallet.manage-wallet
+        [:div.wallet-name "Manage wallets"]
+        [:span.wallet-points
+         [:div.point] [:div.point] [:div.point]]
```

Comment:
This implementation of the ellipsis is not responsive. Try using a single element as an image, svg or even text ("...").